File: AssignMentEaster/nokia_phone.py
while True:
    print("\n")
    print("\n******************")
    print(":WELCOME NEW USER:")
    print("******************")
    print("ABIDO-SHAKERS NOKIA PHONE MENU")

    print(" 1. PHONEBOOK\t2. MESSAGES\t3. CHAT\t4. CALL REGISTER\n5. TONES\t6. SETTINGS\t7. CALL DIVERTS\t8. GAMES\t9.CALCULATOR\n10. REMINDERS\t11.CLOCK\t12.PROFILES\t13.SIM SERVICES\t14. EXIT  ")
    option = int(input("Enter an option: "))

    if option == 14:
        print("Exiting the menu.")
        break


    if option == 1:
        print("1. Search\n 2. Service Nos.'\n 3. Add name\n 4. Erase\n 5. Edit\n 6. Assign tone\n 7. Send b'card\n 8. Options\n 9. Speed dials\n 10. Voice tags\n")
        print()
        sub_option = int(input("Enter your sub-Option: "))
        print()
        
        if sub_option == 1:
            print("Searching...")
        elif sub_option == 2:
            print("Service Nos.'...")
        elif sub_option == 3:
            print("Adding name...")
        elif sub_option == 4:
            print("Erasing...")
        elif sub_option == 5:
            print("Editing...")
        elif sub_option == 6:
            print("Assigning Tone...")
        elif sub_option == 7:
            print("Sending b'card...")
        elif sub_option == 8:
            print("Available options...")
            print(" a. type of view")
            print(" b. memory status")
        elif sub_option == 9:
            print("Speed dialing...")
        elif sub_option == 10:
            print("Opening Voice tags...")
        else:
            print("Choose again or getat!!")

    elif option == 2:
        print("1. Write messages\n 2. Inbox\n 3. Outbox\n 4. Picture messages\n 5. Templates\n 6. Smileys\n 7. Message settings\n 8. Info service\n 9. Voice Mailbox Number\n 10. Service command Editor")
        print()
        sub_choice = int(input("Enter your sub-choice: "))
        print()
        
        if sub_choice == 1:
            print("Writing messages...")
        elif sub_choice == 2:
            print("Viewing inbox...")
        elif sub_choice == 3:
            print("Viewing outbox...")
        elif sub_choice == 4:
            print("Viewing picture messages...")
        elif sub_choice == 5:
            print("Viewing templates...")
        elif sub_choice == 6:
            print("Viewing smileys...")
        elif sub_choice == 7:
            print("Message Settings Menu:")
            print("a. SET")
            print("   i. Message Centre Number")
            print("   ii. Message Sent As")
            print("   iii. Message Validity")
            print("b. Common")
            print("   i. Delivery Reports")
            print("   ii. Reply via Same Centre")
            print("   iii. Character Support")
        elif sub_choice == 8:
            print("Accessing info service...")
        elif sub_choice == 9:
            print("Setting voice mailbox number...")
        elif sub_choice == 10:
            print("Editing service commands...")
        else:
            print("abg exit!!!.")

    elif option == 3:
        print("CHAT")
        chatzone = int(input("Input your chatzone: "))
        if chatzone == 1:
            print("Let's chat...")
        else:
            print("Abeg commot!!!")

    elif option == 4:
        print("1. Missed calls\n 2. Received calls\n 3. Dialed numbers\n 4. Erase recent calls lists\n 5. Show calls duration\n a. Last call duration\n b. All calls duration\n c. Received calls duration\n 6. Dialed calls' duration\n 7. Call timers\n 8. Show call costs\n a. Last call cost\n b. All call cost\n c. Clear counters\n 9. Call cost settings\n a. Call cost limit\n b. Show costs in\n 10. Prepaid credit")
        print()
        call_register = int(input("Enter your callRegister: "))
        print()
        if call_register == 1:
            print("Missed calls...")
        elif call_register == 2:
            print("Received calls...")
        elif call_register == 3:
            print("Dialed numbers...")
        elif call_register == 4:
            print("Erasing recent calls lists...")
        elif call_register == 5:
            print("Showing call durations...")
            print(" a. Last call")
            print(" b. All calls duration")
            print(" c. Received calls duration")
        elif call_register == 6:
            print("Dialed calls' duration...")
        elif call_register == 7:
            print("Call timers...")
        elif call_register == 8:
            print("Showing call costs...")
            print(" a. Last call cost")
            print(" b. All call cost")
            print(" c. Clearing counters")
        elif call_register == 9:
            print("Call cost settings...")
            print(" a. Call cost limit")
            print(" b. Showing call cost")
        elif call_register == 10:
            print("Prepaid credit...")
        else:
            print("Wetin u dey find wey no lost")

    elif option == 5:
        print("1. Ringing tone\n 2. Ringing volume\n 3. Incoming call alert\n 4. Composer\n 5. Message alert tone\n 6. Keypad tones\n 7. Warning and game tones\n 8. Vibrating alert\n 9. Screen saver")
    elif option == 6:
        print("1. Call settings\n 2. Phone settings\n 3. Security settings\n 4. Restore factory")
        print()
        print("Enter your settings:")
        settings = int(input())
        if settings == 1:
            print("1. Call settings...")
            print(" 1. Automatic redial\n 2. Speed dialling\n 3. Call waiting options\n 4. Own number sending\n 5. Phone line in use\n 6. Automatic answer")
        elif settings == 2:
            print("2. Phone settings...")
            print("1. Language\n 2. Cell info display\n 3. Welcome note\n 4. Network selection\n 5. Lights\n 6. Confirm SIM service actions")
        elif settings == 3:
            print("3. Security settings...")
            print("1. PIN code request\n 2. Call barring service\n 3. Fixed dialling\n 4. Closed user group\n 5. Phone security\n 6. Change access codes")
        elif settings == 4:
            print("Restoring factory settings..........")
	else:
            print("Invalid choice")
    elif option == 7:
	print("Call diverted........")
            print("**********************")
    elif option == 8:
        print("Games loading........")
            print("**********************")
    elif option == 9:
        print("Loading Calculator........")
        print("**********************")
    elif option == 10:
        print("Setting up reminders........")
        print("**********************")
    elif option == 11:
        print("Clock settings........")
        print("**********************")
        print()
            print("Enter your clock settings:")
            clock_settings = int(input())
        if clock_settings == 1:
            print("Clock setup")
            print("1. Alarm clock\n 2. Clock settings\n 3. Date setting\n 4. Stopwatch\n 5. Countdown timer\n 6. Auto update of date and time")
    elif choice == 12:
        print("Creating profiles........")
        print("**********************")
    elif choice == 13:
        print("Loading SIM services........")
        print("**********************")
    elif choice == 14:
        print("Exiting... thank you for using the abidoShaker phone menu")
        return
else:
        print("E don do you self rest!!!")

File: semicolon/chi practice/NokiaPhone.java
import java.util.Scanner;

public class NokiaPhone {
public static void main(String... args) {
Scanner scanner = new Scanner(System.in);

System.out.println("Welcome to Nokia Phone Menu!");
int userInput;
int userInput1 = 0;
int userInput2 = 0;
int userInput3 = 0;


String prompt = """
        -------------------
         WELCOME NEW USER
       -------------------
        1 -> PHONEBOOK
        2 -> MESSAGES
        3 -> CALL REGISTER
       
      
        """;


System.out.println( prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();


String p = """
1. Search
2. Service Nos.
3. Add name
4. Erase
5. Edit
6. Assign tone
7. Send b'card
8. Options
    a. Type of view
    b.Memory status
9. Speed dials
10. Voice tags
11. Enter 11 to go back
""";

String  Messages = """
1. Write messages
2. Inbox
3. Outbox
4. Picture messages
5. Templates
6. Smileys
7. Message settings
   
   a. SET
   i) message centre number
   ii) message sent as
   iii) message validity
   
   b. Common
   i) Delivery reports
   ii) Reply via same centre
   iii) Character support

8. Info service
9. Voice mailbox number
10. Service command editor
""";
String  callRegister = """
1. Missed calls
2. Received calls
3. Dialled numbers
4. Erase recent call lists
5. Show call duration
  a. Last call duration
  b. All calls' duration
  c. Received calls' duration
4. Dialled calls' duration
5. Clear timers
6. Show call costs
  a. Last call cost
  b. All calls' cost
  c. Clear counters
7. Call cost settings
  a. Call cost limit
  b. Show costs in
8. Prepaid credit
""";


switch (userInput){
case 1 -> {System.out.println("PHONEBOOK");
          System.out.println(p);
          System.out.println("choose NEXT option");
          userInput1  = scanner.nextInt();}
          
}

 switch (userInput1) {
case 1 -> {System.out.println("Search");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 2 -> {System.out.println("Service Nos.");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
 System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 3 -> {System.out.println("Add name");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 4 -> {System.out.println("Erase");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 5 -> {System.out.println("Edit");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
 System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 6 -> {System.out.println("Assign tone");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
 System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 7 -> {System.out.println("Send b'card");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
 System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 8 -> {System.out.println("Options");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
 System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 9 -> {System.out.println("Speed dials");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 10 -> {System.out.println("Voice tags");
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();}

case 11 -> {
System.out.println();
System.out.println("EMPTY");
System.out.println();
System.out.println();
 System.out.println(prompt);
System.out.println("Select an option:");
userInput = scanner.nextInt();;}
default ->  {System.out.print("e choke wetin u still de find");}
}

switch (userInput){
case 2 -> {System.out.println("MESSAGES");
          System.out.println(Messages);
          System.out.println("choose NEXT option");
          userInput2  = scanner.nextInt();}
}

switch (userInput2) {
case 1 -> {System.out.println("Write messages");}
case 2 -> {System.out.println("Inbox");}
case 3 -> {System.out.println("Outbox");}
case 4 -> {System.out.println("Picture messages");}
case 5 -> {System.out.println("Templates");}
case 6 -> {System.out.println("Smileys");}
case 7 -> {System.out.println("Message settings");}
case 8 -> {System.out.println("Info service");}
case 9 -> {System.out.println("Voice mailbox number");}
case 10 -> {System.out.println("Service command editor");
}
}

switch (userInput){
case 3 -> {System.out.println("CALL REGISTER");
          System.out.println(callRegister);
          System.out.println("choose NEXT option");
          userInput3  = scanner.nextInt();
}
}
switch (userInput3) {
case 1 -> {System.out.println("Missed calls");}
case 2 -> {System.out.println("Recieved calls");}
case 3 -> {System.out.println("Dialled numbers");}
case 4 -> {System.out.println("Erase recent call lists");}
case 5 -> {System.out.println("Clear timers");}
case 6 -> {System.out.println("Show call costs");}
case 7 -> {System.out.println("Call cost settings");}
case 8 -> {System.out.println("prepaid credit");}
}
}
}
